components/colorarea/index.css

```diff
@@ -0,0 +1,64 @@
+:root {
+  --spectrum-colorarea-width: calc(var(--spectrum-global-dimension-size-125) * 16);
+  --spectrum-colorarea-height: var(--spectrum-colorarea-width);
+  --spectrum-colorarea-border-radius: var(--spectrum-alias-border-radius-regular);
+}
+
+%spectrum-ColorControl-hiddenField {
+  opacity: 0.0001;
+  position: absolute;
+  top: 0; left: 0;
+  width: 100%; height: 100%;
+  z-index: 0;
+  margin: 0;
+  pointer-events: none;
+}
+
+%spectrum-ColorControl-handle--focused {
+  /* Bigger handle when focused */
+  width: calc(var(--spectrum-colorhandle-size) * 2);
```

Comment:
`spectrum-colorhandle-size` variable is not defined in this file but in colorhandle component. Why is it still working?  Is it because colorarea depending on colorhandle package, so the user can use a variable defined in depending package?

Reply:
That var is defined in DNA, so it's available here.

There are many other vars in DNA we could be using instead of re-defining them. I'll take a look at that.